<?xml version="1.0"?>
<launch>
  <arg name="mqtt_server_ip"     default="10.23.0.125"/>

  <arg name="agvSn"     default="agv-01"/>

    <node name="scan_republisher" pkg="scan_republisher" type="scan_republisher" output="screen">
        <param name="mqtt_server_ip"    value="$(arg mqtt_server_ip)"/>
        <param name="agvSn"    value="$(arg agvSn)"/>
    

    </node>
</launch>
